tools/build_cdn.js

```diff
@@ -58,6 +64,22 @@ async function buildCDN(options) {
   log("-----");
 }
 
+
+async function buildSRIDigests(shas) {
+  const temp = await fs.readFile("./tools/templates/DIGESTS.md")
```

Comment:
```suggestion
  const temp = fs.readFileSync("./tools/templates/DIGESTS.md")
```

`readFileSync` does exist but not like it matters here